client/components/data/query-share-post-actions/index.jsx

```diff
@@ -7,16 +7,20 @@ import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
-import isFetchingPublicizeShareActions from 'state/selectors/is-fetching-publicize-share-actions';
-import { fetchPostShareActions } from 'state/sharing/publicize/publicize-actions/actions';
+import isFetchingPublicizeShareActionsScheduled from 'state/selectors/is-fetching-publicize-share-actions-scheduled';
+import isFetchingPublicizeShareActionsPublished from 'state/selectors/is-fetching-publicize-share-actions-published';
+import { fetchPostShareActionsScheduled, fetchPostShareActionsPublished } from 'state/sharing/publicize/publicize-actions/actions';
 
 class QuerySharePostActions extends Component {
 	static propTypes = {
 		siteId: PropTypes.number,
 		postId: PropTypes.number,
+		status: PropTypes.string,
 		// Connected props
-		isRequesting: PropTypes.bool.isRequired,
-		fetchPostShareActions: PropTypes.func.isRequired,
+		isRequestingScheduled: PropTypes.bool.isRequired,
```

Comment:
I would not mark these as `isRequired` because they are passed from `connect` and don't have to be specified when using the component.